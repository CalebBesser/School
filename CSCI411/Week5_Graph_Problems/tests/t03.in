5 6
0 3
1 2
2 0
3 4
3 1
4 2
printGraph
isFlower
