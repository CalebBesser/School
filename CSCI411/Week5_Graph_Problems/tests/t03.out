Adjacency lists of the graph are:
0: 3 
1: 2 
2: 0 
3: 4 1 
4: 2 
The graph is NOT a flower-graph.
